src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component } from '@angular/core';
+import { CHILDREN } from './mock-data';
+import { Child } from '../child';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.scss']
+})
+export class DashboardComponent {
+  santaList = CHILDREN;
+  currentChild = new Child('', '', false);
+
+  saveChild() {
+
+    // boolean to tell if adding new child or saving existing child info
+    let alreadyExists = false;
+
+    for (let child of this.santaList) {
```

Comment:
Look at comments from day 4 PR. Try using the array iterator functions that are built in, ie `this.santaList.some()`